Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,299 @@
+
+- [VOID]   
+  Click this button to disable the transaction for the Actindo database. For detailed information, see [Void authorization transaction](../Operation/01_ManageAuthorizations.md#void-authorization-transaction).
+
+
+The list displays all authorization transactions. Depending on the payment service provider, the displayed columns vary. All fields are read-only. The following standard columns are available:
+
+- *External ID*   
+   ID given by the payment service provider.
+
+- *Original reference*   
+   Reference to the authorization transaction. You can use this reference to follow up one payment over the whole process.
+
+- *Transaction date*   
+   Date on which the transaction has been created by the payment service provider.
+
+- *Status*   
+   Status text.   
+
+- *Status ID*   
+   Identifier for the status. The following statuses are available, each prefixed by the ID in the list below. You can use the *Status ID* to filter and sort the list. 
+-   - **1 - Preparing**  
+        The transaction has been created, but is not yet transferred.
+    - **2 - Unconfirmed**   
+        A refund is reserved but not yet released by an Actindo user. 
+    - **3 - Success**  
+        Actindo has got the response from the service payment provider that the transfer was successful.
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the status information field may provide further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void an authorization transaction](01_ManageAuthorizations.md#void-an-authorization-transaction).
+
+- *Connection*   
+   Name of the Connection. Note that this is the customer-specific name that has been given the connection. For detailed information, see [Establish connections to payment service providers (PSP)](../Integration/01_EstablishConnection.md).
+
+- *Created by*   
+   User name of the person or API who has created the transaction. Can be a real user or an API user.
+
+- *Modified by*   
+   User name of the person or API who has changed the transaction. Can be a real user or an API user.
+
+- *ID*   
+   Actino unique identifier of the transaction.   
+     
+- *Type*   
+   Kind of the displayed transaction, in this case an authorization transaction.  
+
+- *Connection ID*   
+   Connection identification number. The ID number is automatically assigned by the system.    
+
+- *Created At*   
+   Date and time, on which the payment service provider has created the transaction. 
+
+- *Modified At*   
+   Date and time, on which the payment service provider has modified the transaction.  
```

Comment:
```suggestion
   Date and time on which the payment service provider has modified the transaction.  
```
Alternativ: Date and time of the last modification.

Reply:
Done